topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/data-library.yaml

```diff
@@ -13,10 +13,11 @@ items:
     - name: 'DOI: 10.5281/zenodo.1560645'
       description: latest
       items:
-      - url: https://zenodo.org/api/files/463f6b73-0c42-4be4-8b52-181a8e093abd/mouse_kidney_cut.ibd
-        src: url
-        ext: ibd
-        info: https://doi.org/10.5281/zenodo.1560645
+# Commented until composite dataset upload is possible (https://github.com/galaxyproject/galaxy/issues/8909)
```

Comment:
Can you need to indent it as the content? Travis error: `16:1      warning  comment not indented like content  (comments-indentation)`